### PR TITLE
Update NonlinearCircuits.

### PR DESCRIPTION
**Pull request by rl2939**
Update NonlinearCircuits and adds the module Splish.

### CODE REVIEW

**Comment by dromer** [comment]
The SVGs have been renamed which requires us to modify the code in `src/custom/dep.cpp`

File: src/custom/dep.cpp [hunk 1]
```diff
@@ -330,19 +330,20 @@ static const struct {
     { kModeMyth, "/myth-modules/res/Molphar.svg", {}, -1 },
     // CC0-1.0
     { kModeNonlinearcircuits, "/nonlinearcircuits/res/BOOLs2.svg", {}, -1 },
```
Comment (dromer) [inline src/custom/dep.cpp:332]: It seems that `BOOLs2.svg` has been deleted?

Comment (rl2939) [reply src/custom/dep.cpp]: Fixed.

**Comment by dromer** [comment]
Ok my bad, it seems that Splish has a dark panel by default :roll_eyes:

**Pull request merged**